package.json

```diff
@@ -12,12 +13,16 @@
     "type": "git",
     "url": "git+https://github.com/xwp/block-scaffolding-wp.git"
   },
+  "engines": {
+    "node": ">=10",
+    "npm": ">=6.9"
+  },
   "scripts": {
     "postinstall": "composer install",
     "dev": "wp-scripts start",
     "build": "wp-scripts build",
     "lint": "npm-run-all --parallel lint:*",
-    "lint:js": "eslint js/src",
```

Comment:
Maybe also:

```diff
diff --git a/package.json b/package.json
index 95fef9a..7617da5 100755
--- a/package.json
+++ b/package.json
@@ -19,6 +19,7 @@
   },
   "scripts": {
     "postinstall": "composer install",
+    "check-engines": "wp-scripts check-engines",
     "dev": "wp-scripts start",
     "build": "wp-scripts build",
     "lint": "npm-run-all --parallel lint:*",
```

...and [like Gutenberg](https://github.com/WordPress/gutenberg/blob/056d7c108ac968363b7a747660ea50b13069f87a/package.json#L182), possibly:
```
"predev": "npm run check-engines",
```

Reply:
That's a great suggestion. I wanted to keep this changeset to a minimum which is why I didn't include it yet. Gutenberg uses the `check-enginges` helper [in several places](https://github.com/WordPress/gutenberg/blob/6e33ffb32fb5711ebd13e179f62e1b4209549d3e/package.json#L173) so we would need to make decisions around that, too.